### PR TITLE
Create env_issue_template.md

### LINKED ISSUE #6: Feature: Refactor GitHub Issue templates
**Is your feature request related to a problem? Please describe.**
`Bug Report` template is duplicated. `Custom` one - don't even know what it is.

**Describe the solution you'd like**

- Remove duplicated templates,
- Remove `Custom` template,
- Add `Test Request` (To Be Consulted),
- Add `Refactor Request` (To Be Consulted),

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by marekksiazek**
Add new issue template with environment.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/env_issue_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Environment issue
+about: Use this template to report problem with environment setting or installation
+title: "[DATE]: [FEATURE NAME]"
+labels: environment, settings
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/env_issue_template.md]: We don't have such labels, please remove.

File: .github/ISSUE_TEMPLATE/env_issue_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: Environment issue
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/env_issue_template.md]: Make "issue" word "Issue".

File: .github/ISSUE_TEMPLATE/env_issue_template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+about: Use this template to report problem with environment setting or installation
+title: "[DATE]: [FEATURE NAME]"
+labels: environment, settings
+assignees: Strayker
+---
+
+**Describe the problem** <br/>
+_Here write your problem with env_
+
+**What env do you have?**
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/env_issue_template.md]: Add environments options list here: dev, test, prod.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Comment by StraykerPL** [comment]
Please update branch.

**Review by marekksiazek — COMMENTED** [review]
Corrected

File: .github/ISSUE_TEMPLATE/env_issue_template.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+name: Environment Issue
+about: Use this template to report problem with environment setting or installation
+title: "[DATE]: [FEATURE NAME]"
+labels: settings
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/env_issue_template.md]: We don't have "settings" label, please remove

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Review by marekksiazek — COMMENTED** [review]
Delete labels

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**